concordium-std/src/impls.rs

```diff
@@ -970,6 +982,7 @@ where
             offset: 8, // Items in a map always start with the set prefix which is 8 bytes.
         };
         // Unwrapping is safe when only using the high-level API.
+        // TODO: This is inefficient. There's no need to allocate the entry.
```

Comment:
I am not sure I understand this TODO.

Reply:
Do you want to make StateRef load the values lazily?

Reply:
I removed the todo.